src/Components/Navbar.tsx

```diff
@@ -0,0 +1,81 @@
+import { HomeOutlined, PlusOutlined, TeamOutlined, UserOutlined, LogoutOutlined } from "@ant-design/icons";
+import type { MenuProps } from "antd";
+import { Menu } from "antd";
+import React, { useContext, useState } from "react";
+import styled from "@emotion/styled";
+import { FormattedMenu } from "./Style";
+import { useGlobalContext } from './UserContext'
+
+
+const HomeIcon = styled(HomeOutlined)(() => ({
+  //color: "blue",
+  ':hover': {
+    a:{
+      color:'red'
+    }
+}, // ez miert nem mukodik???
+}));
+
+const items: MenuProps["items"] = [
+  
+  {
+    label: (
+      <a href="/" rel="noopener noreferrer">
+        Home
+      </a>
+    ),
+    key: 'home',
+    icon: <HomeIcon/>
+  },
+  {
+    label: (
+      <a href="/create" rel="noopener noreferrer">
+        Create new journey
+      </a>
+    ),
+    key: 'create',
+    icon: <PlusOutlined />
+  },
+  {
+    label: (
+      <a href="/about" rel="noopener noreferrer">
+        About
+      </a>
+    ),
+    key: 'about',
+    icon: <TeamOutlined />
+  },
+  {label: (
+      <a href="/login" rel="noopener noreferrer">
+        Login
+      </a>
+    ),
+    key: 'login',
+    icon: <UserOutlined />
+    
+  },
+];
+
+const Navbar: React.FC = () => {
+  const [current, setCurrent] = useState("mail"); // ant desinbol jon ? funkcio
```

Comment:
Megadtal minden itemhez egy key: string erteket, majd amikor valamelyikre kattintasz, akkor a statet beallitod erre a string ertekre. Ennek a segitsegevel tudod vizualisan is megkulonboztetni a jelenleg aktiv menupontot.

Reply:
Tippre a Formattedmenu belsejeben ez le is van kezelve.